docs/basics/ExampleTodoList.md

```diff
@@ -9,116 +9,85 @@ This is the complete source code of the tiny todo app we built during the [basic
 ```js
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
 import { Provider } from 'react-redux'
-import App from './containers/App'
+import { createStore } from 'redux'
 import todoApp from './reducers'
+import App from './components/App'
 
 let store = createStore(todoApp)
```

Comment:
Should this be `todoStore`?
https://github.com/rackt/redux/pull/1285/files#diff-e9a37286e0b5c53a133d6291766a478fR20